### ISSUE TITLE
DropDownList.cs: suggested change of the function UpdateSelected()

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#123]
<sup>Issue created by carlos helder as [Bitbucket Issue #&#8203;87](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/87) on 2016.07.04 16:24.</sup>
Hi! I would actually remove the following line:

```c#

_panelItems[i].btnImg.color = (_selectedIndex == i) ? _mainButton.btn.colors.highlightedColor : new Color(0, 0, 0, 0);
```

Or at least, remove  
```c#

: new Color(0, 0, 0, 0);
```
 because it removes the highlighting capability when the cursor is on any item. Anyway I'm curious: why is that code like that?

Thank you for sharing this awesome set of UI code!

Carlos

**Comment by SimonDarksideJ** [comment]
<sup>On [2016.07.14 17:57](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/87#comment-34323162), @SimonDarksideJ commented:</sup>
Likely the original author found it useful.  I'll add an option to turn off that highlighting rather than just remove it so it doesn't break backwards compatibility.
I'll also add a field and remove the hard coded color value for good measure as well.

Thanks for spotting that.

**Comment by SimonDarksideJ** [comment]
<sup>On [2016.07.14 18:49](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/87#comment-34323161), @SimonDarksideJ commented:</sup>
Added TextPic component from #89
Added Override Highlighted option to drop down to fix #87

Referenced by 08571b1f36fa

**Comment by SimonDarksideJ** [comment]
<sup>On 2016.07.14 18:49 @SimonDarksideJ modified issue:</sup>
**status** changed `new` → `resolved`

**Comment by SimonDarksideJ** [comment]
<sup>On [2016.07.14 18:49](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/87#comment-34323160), @SimonDarksideJ commented:</sup>
Added TextPic component from #89
Added Override Highlighted option to drop down to fix #87

Referenced by 08571b1f36fa
